README.md

```diff
@@ -118,6 +118,18 @@ client = Kubeclient::Client.new 'https://localhost:8443/api/' , 'v1',
                                 auth_options: auth_options
 ```
 
+If your system is already configured with application default credential as described
+[here](https://developers.google.com/identity/protocols/application-default-credentials)
+then you can use the following:
+
+```ruby
+auth_options = {
+  default_credentials: true
```

Comment:
I would name this use_default since its boolean. It seems more consistent with the other auth keys that all contain string values. Also maybe use_default_gce or use_gce is better since it's currently only gce?
Naming is hard :ram: